charts/wildfly/README.md

```diff
@@ -141,6 +148,7 @@ If the application image has been built by another mechanism, you can skip the b
 | `build.s2i.galleonLayers` | *Deprecated* A list of layer names to compose a WildFly server. If specified, `build.s2i.featurePacks` must also be specified. | - | The value can be be either a `string` with a list of comma-separated layers or an array where each item is a layer - [WildFly S2I documentation](https://github.com/wildfly/wildfly-s2i) |
 | `build.s2i.kind` | Determines the type of images for S2I Builder and Runtime images (`DockerImage`, `ImageStreamTag` or `ImageStreamImage`) | `DockerImage` | (OKD Documentation](https://docs.okd.io/latest/cicd/builds/build-strategies.html#builds-strategy-s2i-build_build-strategies) |
 | `build.s2i.runtimeImage` | WildFly S2I Runtime image | [quay.io/wildfly/wildfly-runtime-jdk11:latest](https://quay.io/repository/wildfly/wildfly-runtime-jdk11) | [WildFly S2I documentation](https://github.com/wildfly/wildfly-s2i) |
+| `build.sourceSecret`|Name the secret containing the credentials to login to git source reposiory | - | The secret must exist in the same namespace or the chart will fail to install - [OpenShift documentation](https://docs.okd.io/latest/cicd/builds/creating-build-inputs.html#builds-manually-add-source-clone-secrets_creating-build-inputs) |
```

Comment:
s/Name the secret containing the credentials/Name of the secret containing the credentials

Reply:
done